actors/build.gradle.kts

```diff
@@ -0,0 +1,6 @@
+val gdxVersion: String by rootProject
```

Comment:
Is there a way of making it global or accessing it without the variable redefined over and over?

Reply:
Hello, unfortunately I didn't find any way to apply it to plugins, but I did get a way for the rest of them!

Reply:
Alright, thanks.